lib/denylist/http.go

```diff
@@ -121,7 +121,8 @@ func deleteDenylistEntry(response http.ResponseWriter, request *http.Request, de
 	}
 	_, exists := denylist.Load(id)
 	if !exists {
-		response.WriteHeader(http.StatusNoContent)
+		log.Log.Warnw("Attempted removal of non-existant denylist entry", "id", id, "error", err.Error())
```

Comment:
There's no `err` value here. I think that might be causing the build issue. Also, small typo.
```suggestion
		log.Log.Warnw("Attempted removal of non-existent denylist entry", "id", id)
```

Reply:
I missed that -- thanks!